File: swagger/apis/staff_api.py
from .base_api import BaseApi

from ..models.get_staff_request import GetStaffRequest
from ..models.get_staff_response import GetStaffResponse
from ..models.get_staffpermissions_request import GetStaffpermissionsRequest
from ..models.get_staff_permissions_response import GetStaffPermissionsResponse

class StaffApi(BaseApi):
	def __init__(self, site_id, authorization):
		super().__init__('staff', site_id, authorization)

	def get_staff(self, request):
		"""
		Get staff members at a site.
		"""

		url = self.get_fullpath('staff')
		return self.client.get(url, self.site_id, self.authorization, request, GetStaffResponse)

	def get_staffpermissions(self, request):
		"""
		Get configured staff permissions for a staff member.
		"""

		url = self.get_fullpath('staffpermissions')
		return self.client.get(url, self.site_id, self.authorization, request, GetStaffPermissionsResponse)

File: swagger/apis/StaffApi.php
<?php

namespace App\Mindbody\Apis;

use App\Mindbody\Models\GetStaffRequest;
use App\Mindbody\Models\GetStaffResponse;
use App\Mindbody\Models\GetStaffPermissionsRequest;
use App\Mindbody\Models\GetStaffPermissionsResponse;

class StaffApi extends BaseApi
{
	/**
	 * Constructor
	 */
	public function __construct($siteId, $authorization)
	{
		parent::__construct('staff', $siteId, $authorization);
	}

	/**
	 * Get staff members at a site.
	 */
	public function getStaff($request)
	{
		$url = $this->getFullPath('staff');

		return $this->client->get($url, $request, GetStaffResponse::class);
	}

	/**
	 * Get configured staff permissions for a staff member.
	 */
	public function getStaffPermissions($request)
	{
		$url = $this->getFullPath('staffpermissions');

		return $this->client->get($url, $request, GetStaffPermissionsResponse::class);
	}

}
